package.json

```
@@ -7,6 +7,16 @@
   "es2015": "./dist/esm/index.js",
   "types": "./dist/types/index.d.ts",
   "sideEffects": false,
+  "exports": {
+    ".": {
+      "node": "./dist/cjs/index.js",
+      "default": "./dist/esm5/index.js"
+    },
+    "./operators": {
+      "node": "./dist/cjs/operators/index.js",
+      "default": "./dist/dist/esm5/operators/index.js"
+    }
+  },
```

FYI this is a breaking change as it locks down the public interface of accessible deep imports. At the very least you probably want an explicit entry for the `package.json`

```suggestion
    },
    "./package.json": "./package.json"
  },
```